src/pages-and-resources/discussions/data/slice.js

```diff
@@ -21,6 +21,8 @@ const slice = createSlice({
     selectedAppId: null,
     status: LOADING,
     saveStatus: SAVED,
+    // ValidationError is the Flag that represents a form is validate or not.
```

Comment:
can we change the comment to valid or not* instead of validate

Reply:
sure